lib/elixir/pages/Typespecs.md

```diff
@@ -139,6 +139,13 @@ Specifications can be overloaded just like ordinary functions.
     @spec function(integer) :: atom
     @spec function(atom)    :: integer
 
+You can also name your arguments in a typespec using `arg_name :: arg_type` syntax.
+This is particularly useful as a way to differentiate multiple arguments
+of the same type (or multiple elements of the same type in a type def):
+
+    @spec new_date(year :: integer, month :: integer, day :: integer)
```

Comment:
This spec is invalid because it's missing the return value. I suggest using one of the named arguments in the return value to show they can be used in multiple places.